jobs/iptables-manager/templates/bin/iptables-manager_ctl.erb

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+export JOB_NAME='iptables-manager'
+export HOME=/var/vcap
+export LOG_DIR=$HOME/sys/log/$JOB_NAME
+export RUN_DIR=$HOME/sys/run/$JOB_NAME
```

Comment:
No. These come from bosh-helpers and didnt want to include this package as i found that blueprint bosh release has its own bosh-helpers package and so could other services which could clash with package name. Hence setting explicitly those env variables which are set from bosh-helpers to avoid including that entire package.